sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update_slm.hpp

```diff
@@ -0,0 +1,720 @@
+//==-------atomic_update_slm.hpp - DPC++ ESIMD on-device test --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "../../esimd_test_utils.hpp"
+
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <sycl/ext/intel/esimd.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::intel::esimd;
+using namespace sycl::ext::intel::experimental::esimd;
```

Comment:
Is this really needed? The test verifies public/non-experimental API
```suggestion
```

Reply:
Done.